test/publish_signal_integration_test.go

```diff
@@ -0,0 +1,114 @@
+// Copyright 2024 Searis AS
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package test
+
+import (
+	"context"
+	"strings"
+	"testing"
+
+	clarify "github.com/clarify/clarify-go"
+	"github.com/clarify/clarify-go/automation"
+	"github.com/clarify/clarify-go/fields"
+	"github.com/clarify/clarify-go/views"
+	"golang.org/x/text/cases"
+	"golang.org/x/text/language"
+)
+
+const (
+	keyExampleName    = "clarify/clarify-go/example/name"
+	keyExamplePublish = "clarify/clarify-go/example/publish"
+	exampleName       = "save_signals"
+	annotationTrue    = "true"
+	transformVersion  = "v2"
+)
```

Comment:
Move constants specific to the test to be defined inside the tests, so they don't pollute the package namespace.

Reply:
Feel free to sanitize/change the values! E.g. if you add a prefix, than `name` and `publish` should be sufficient as annotation keys.